### ISSUE TITLE
tslint.json location in rule?

### ISSUE THREAD

**Issue by flapenguin** [palantir/tslint#4568]
Any way to get `tslint.json` location inside a rule? I'm passing some paths to my linter and it doesn't work very good with per-folder configurations.

```sh
tslint -p ./subfolder
```

`./subfolder/tslint.js`
```js
module.exports = {
  "rules": {
    "my-custom-rule": {
      // meaning ./folder/foo.bar.js
      // process.cwd() inside rule is './', not './subfolder'
      "options": ["./foo.bar.js"]
    }
  }
}
```

**Comment by Akuukis** [comment]
@yantrab create another `tslint.json` file in your subfolder and it will take precedence over the one in root folder. No need to do anything else.

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is being deprecated** and no longer accepting pull requests for major new changes or features. See #4534. 😱

If you'd like to see this change implemented, you have two choices:
* **Recommended:** Check if this is available in [ESLint](https://eslint.org) + [typescript-eslint](https://typescript-eslint.io) ✅
* _Not Recommended: Fork TSLint locally_ 🤷‍♂️

👋 It was a pleasure open sourcing with you!

_If you believe this message was posted here in error, please comment so we can re-open the issue!_

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
